charactersheet/charactersheet/templates/dm/encounter_sections/treasure_section.tmpl.html

```diff
@@ -0,0 +1,1338 @@
+     role="dialog"
+     data-bind="modal: {
+      onclose: addModalFinishedClosing
+    }">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button"
+                class="close"
+                data-dismiss="modal"
+                aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+        <h4 class="modal-title"
+            id="addNpcLabel">Add more loot to the treasure horde.</h4>
+    </div>
+      <div class="modal-body">
+        <form class="form-horizontal">
+
+          <div class="form-group">
+            <div class="col-sm-12">
+              <div class="btn-group btn-group-justified" role="group">
+                <label class="btn btn-default btn-group-padding"
+                       data-bind="css: { active: itemType() == 'armor'}">
+                  <img class="action-bar-icon" src="/images/misc_icons/breastplate.svg"></img>
+                  <input type="radio" class="hide-block"
+                    data-bind="checked: itemType, click: setTreasure"
+                    name="armor" value="armor"/>Armor
+                </label>
+                <label class="btn btn-default btn-group-padding"
+                       data-bind="css: { active: itemType() == 'coins'}">
+                 <img class="action-bar-icon" src="/images/misc_icons/coins.svg"></img>
+                  <input type="radio" class="hide-block" data-bind="checked: itemType, click: setTreasure"
+                         name="coins" value="coins"/>Coins
+                </label>
+                <label class="btn btn-default btn-group-padding"
+                       data-bind="css: { active: itemType() == 'item'}">
+                  <img class="action-bar-icon" src="/images/misc_icons/drink-me.svg"></img>
+                  <input type="radio" class="hide-block" data-bind="checked: itemType, click: setTreasure"
+                         name="item" value="item"/>Item
+                </label>
+                <label class="btn btn-default btn-group-padding"
+                       data-bind="css: { active: itemType() == 'magicItem'}">
+                  <img class="action-bar-icon" src="/images/misc_icons/fairy-wand.svg"></img>
+                  <input type="radio" class="hide-block" data-bind="checked: itemType, click: setTreasure"
+                       name="magicItem" value="magicItem"/>Magic Item
+                </label>
+                <label class="btn btn-default btn-group-padding"
+                       data-bind="css: { active: itemType() == 'weapon'}">
+                  <img class="action-bar-icon" src="/images/misc_icons/broadsword.svg"></img>
+                  <input type="radio" class="hide-block" data-bind="checked: itemType, click: setTreasure"
+                        name="weapon" value="weapon"/>Weapon
+                </label>
+              </div>
+            </div>
+         </div>
+
```

Comment:
In order to resolve the error I'm getting in the console after adding new items you could add something like this:

```html
<!-- ko if: blankItem() -->
... Modal template here.
<!-- /ko -->
```